testdata/d2compiler/TestCompile/edge_arrowhead_fields.exp.json

```diff
@@ -253,6 +253,8 @@
         "index": 0,
         "minWidth": 0,
         "minHeight": 0,
+        "fromTableRowIndex": 0,
```

Comment:
is there a test where this is non 0?

Reply:
good catch.
I added the sql tables test in the previous PR, but it seems it got lost during the rebase 🤔